driver/pparse.mli

```diff
@@ -28,6 +28,19 @@ type 'a ast_kind =
 | Structure : Parsetree.structure ast_kind
 | Signature : Parsetree.signature ast_kind
 
+val clear_ppx : unit -> unit
+(** Forget about all installed rewriters: in-process and external ones *)
+
+val clear_in_process_ppxs: unit -> unit
```

Comment:
As I've explained before, indiscriminately removing all in-process ppxs is an operation that does not seem useful and it will actively break ocamlfind if it ever starts installing in-process ppxs (which is something I think is quite useful).